### ISSUE TITLE
vSphere.VMware.SsoAdmin - some improvements

### ISSUE THREAD

**Issue by philiprusinov** [vmware/PowerCLI-Example-Scripts#420]
Hello,
Could you please implement "-SecondaryUrl" on Get-IdentitySource cmdlet, as you did it on  Add-LDAPIdentitySource. This will give possibility to pipe Get-IdentitySource | Set-LDAPIdentitySource -Certificates and update certificates on environments, where 2 DCs are in use. Currently Get-IdentitySource is querying only the primary DC, so update certificate is not possible if in the current configuration you have 2 DCs.
Could you please also add Remove-LDAPIdentitySource cmdlets?
Thanks!

**Comment by dmilov** [comment]
I'll prepare new version of the module that includes this request

**Comment by dmilov** [comment]
* Added `FailoverUrl` property to the ActiveDirectoryIdentitySource result

```powershell
Get-IdentitySource -External

Type                   : ActiveDirectory
Alias                  : sof-powercli-qe
AuthenticationType     : password
AuthenticationUsername : administrator@sof-powercli-qe
FriendlyName           : sof-powercli-qe
PrimaryUrl             : ldap://sof-powercli-qe.eng.vmware.com:389
FailoverUrl            :
UserBaseDN             : CN=Users,DC=sof-powercli-qe,DC=eng,DC=vmware,DC=com
GroupBaseDN            : CN=Users,DC=sof-powercli-qe,DC=eng,DC=vmware,DC=com
Name                   : sof-powercli-qe.eng.vmware.com
```

It is possible to filter out the results of Get-IdentitySource by this property the following way
```powershell
Get-IdentitySource -External | Where-Object {$_.FailoverUrl -like '<search pattern>'}
```

* Added `Remove-IdentitySource` cmdlet that enables removing of identity sources, e.g.

```powershell
Get-IdentitySource -External | Remove-IdentitySource
```

Fixed in version 1.2.3 available on GitHub and PSGallery

**Comment by philiprusinov** [comment]
Thank you Dimitar!
